app/controllers/events.server.controller.js

```diff
@@ -0,0 +1,49 @@
+const Event = require('mongoose').model('Event');
+
+function getErrorMessage(err) {
+  let message = '';
+
+  if (err.code) {
```

Comment:
If I click the "Create Event" button on an empty form, a validation error accrues first (Minimum of people must be less than the maximum) instead of "Type is required". Just as a minor flaw...

Reply:
This is because the numberOfPeopleTo validator is run before required specification on the eventType field. Once values are provided for the number of people, the Type is required message displays.